Source/Common/IGListDiff.h

```diff
@@ -59,4 +70,38 @@ FOUNDATION_EXTERN IGListIndexPathResult *IGListDiffPaths(NSInteger fromSection,
                                                          NSArray<id<IGListDiffable>> *_Nullable newArray,
                                                          IGListDiffOption option);
 
+/**
+ Creates a diff using indexes between two collections.
+
+ @param oldArray The old objects to diff against.
+ @param newArray The new objects.
+ @param option   An option on how to compare objects.
+ @param behavior Comparison behaviors.
+
+ @return A result object containing affected indexes.
+ */
+FOUNDATION_EXTERN IGListIndexSetResult *IGListDiffWithBehavior(NSArray<id<IGListDiffable>> *_Nullable oldArray,
```

Comment:
can we keep the same names for these new functions? `IGListDiff` and `IGListDiffPaths` ?